### DISCUSSION TITLE
Feature Request: Compiled binaries for Arm64 Linux

### DISCUSSION THREAD

**Discussion by toastmanAu** [nervosnetwork/neuron#2794] [Ideas]
Can we get official releases of Neuron for Arm64 Linux?

**Comment by Keith-CY** [comment]
> Can we get official releases of Neuron for Arm64 Linux?

Hi @toastmanAu 

We cannot give a definite answer at the moment, but we have listed it in our task to investigate compiling Neuron for Arm64 Linux.

It is feasible based on the document(https://www.electron.build/cli.html), but requires further validation and testing.

We'll try to build it for internal testing and hope to include it in the version after next.

**Reply by toastmanAu** [reply]
Thanks @Keith-CY. Appreciate you guys looking into it. I've got a build running on an ubuntu 22.04 sbc currently but it requires a standalone node to be running to function. i didn't compile my own indexer though and used the x86 binary which is probably why that component isn't working for me. An official release would be great.

**Comment by Keith-CY** [comment]
Hi @toastmanAu we are trying to release Neuron for Arm64 Linux but it was blocked by https://github.com/Magickbase/neuron-public-issues/issues/256#issuecomment-1772017801
We will keep an eye on it, and updates can be tracked by https://github.com/Magickbase/neuron-public-issues/issues/256
